imap_processing/codice/codice_l1a.py

```diff
@@ -36,19 +310,48 @@ def codice_l1a(packets: list[space_packet_parser.parser.Packet]) -> str:
         The path to the CDF file that was created
     """
     # Group data by APID and sort by time
+    print("Grouping the data by APID")
     grouped_data = group_by_apid(packets)
+    grouped_data = {1154: [], 1156: []}  # Temporary during development
```

Comment:
It might be helpful to use the named values from the Enums you created instead of these integers. ` CODICEAPID.COD_NHK.value` or something similar.